docs/platform/howto/byoc/create-custom-cloud.rst

```diff
@@ -0,0 +1,138 @@
+
+About creating a custom cloud
+-----------------------------
+
+If you enabled BYOC and you have the administrator's role in your organization, you can create a custom cloud using `Aiven Console <https://console.aiven.io/>`_.
+
+Limitations
+'''''''''''
+
+* Administrator's role is required for creating custom clouds.
+* You need to use AWS as a cloud provider.
+* You need to use the :ref:`BYOC standard deployment <byoc-standard>` as a deployment model for your custom cloud.
+
+Prerequisites
+-------------
+
+* Administrator's role for your Aiven organization
+* BYOC feature enabled from `sales@Aiven.io <mailto:sales@Aiven.io>`_
+* Access to `Aiven Console <https://console.aiven.io/>`_
+
+Create a custom cloud
+---------------------
+
+Navigate to BYOC in Aiven Console
+'''''''''''''''''''''''''''''''''
+
+1. Log in to `Aiven Console <https://console.aiven.io/>`_ as an administrator.
+2. From the left sidebar, select **Bring you own cloud**.
+3. In the **Bring you own cloud** view, select **Create custom cloud**.
+
+Generate an infrastructure template
+'''''''''''''''''''''''''''''''''''
+
+In this step, an infrastructure-as-code (IaC) template is generated. For that purpose, AWS CloudFormation or Terraform can be used.
+
+In the **Create custom cloud** wizard, proceed as follows:
+
+1. Specify the following:
+
+   * Cloud name
+   * Cloud provider
+   * Region
+   * CIDR
+
+     Aiven needs CIDR for the `CIDR block of the VPC <https://docs.aws.amazon.com/vpc/latest/userguide/vpc-cidr-blocks.html>`_ that will be created in your AWS account.
+
+     Specify inbound rules with the CIDR block notation, for example: 200.1.2.3/32 (allowing 200.1.2.3 as a single address), 0.0.0.0/0 (allowing traffic from anywhere), or 100.1.0.0/16 (allowing traffic from 100.1..).
+
+     Note the following:
+
+     * To create VPC peerings with that VPC, choose a CIDR block that doesn't overlap with CIDR blocks of peer VPCs.
+     * CIDR block needs be large enough so that, after splitting it into per-region subnets, each subnet has enough addresses to fit required services.
+   * :ref:`Deployment model <byoc-deployment>`
+
+     The deployment model determines how resources within your Aiven organization are arranged. It also imposes the method of connectivity between Aiven's control plane and networks under your cloud provider account.
+
+2. Select **Next**.
+   
+.. topic:: Result
+
+    Your infrastructure template gets generated based on your inputs. You can view, copy, or download it.
+
+Acquire ``Role ARN``
+''''''''''''''''''''
+
+Role ARN (Amazon Resource Name) is an `identifier of the role <https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles.html>`_ created when running the infrastructure template in your AWS account. Aiven uses Role ARN to `assume the role <https://docs.aws.amazon.com/STS/latest/APIReference/API_AssumeRole.html>`_ and run operations such as creating VMs for service nodes in your BYOC account.
```

Comment:
We haven't specified anywhere above that this beta instruction is available now for AWS only. I believe it makes sense to state that more clearly